<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1270</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/cryptonote</normaloff>:/images/cryptonote</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #212529;
color: #ccc;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="WalletGui::WelcomeFrame" name="m_welcomeFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::OverviewFrame" name="m_overviewFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::SendFrame" name="m_sendFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::ReceiveFrame" name="m_receiveFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::TransactionsFrame" name="m_transactionsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::AddressBookFrame" name="m_addressBookFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::MessagesFrame" name="m_messagesFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::SendMessageFrame" name="m_sendMessageFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::DepositsFrame" name="m_depositsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::BankingFrame" name="m_bankingFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::BankingFrame2" name="m_bankingFrame2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="m_overviewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/home-icon.png</normaloff>:/icons/home-icon.png</iconset>
   </property>
   <property name="text">
    <string>OVERVIEW</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_sendAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-send.png</normaloff>:/icons/icon-send.png</iconset>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_receiveAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-keys.png</normaloff>:/icons/icon-keys.png</iconset>
   </property>
   <property name="text">
    <string>KEYS</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_transactionsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-tx.png</normaloff>:/icons/icon-tx.png</iconset>
   </property>
   <property name="text">
    <string>TRANSACTIONS</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_exitAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_addressBookAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-address.png</normaloff>:/icons/icon-address.png</iconset>
   </property>
   <property name="text">
    <string>ADDRESS BOOK</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_createWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create wallet</string>
   </property>
  </action>
  <action name="m_openWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open wallet</string>
   </property>
  </action>
  <action name="m_encryptWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Encrypt wallet</string>
   </property>
  </action>
  <action name="m_changePasswordAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
   <property name="toolTip">
    <string>Change password</string>
   </property>
  </action>
  <action name="m_aboutCryptonoteAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="m_aboutQtAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="m_backupWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backup wallet</string>
   </property>
  </action>
  <action name="m_startOnLoginAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start on system login</string>
   </property>
  </action>
  <action name="m_minimizeToTrayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
  <action name="m_closeToTrayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close to tray</string>
   </property>
  </action>
  <action name="m_messagesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-messages.png</normaloff>:/icons/icon-messages.png</iconset>
   </property>
   <property name="text">
    <string>MESSAGES</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_sendMessageAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-sendmsg.png</normaloff>:/icons/icon-sendmsg.png</iconset>
   </property>
   <property name="text">
    <string>SEND MESSAGE</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_importKeyAction">
   <property name="text">
    <string>Import GUI key</string>
   </property>
  </action>
  <action name="m_depositsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-deposit.png</normaloff>:/icons/icon-deposit.png</iconset>
   </property>
   <property name="text">
    <string>DEPOSITS</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_bankingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/icon-deposit.png</normaloff>:/icons/icon-deposit.png</iconset>
   </property>
   <property name="text">
    <string>BANKING</string>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="m_resetAction">
   <property name="text">
    <string>Reset wallet</string>
   </property>
   <property name="toolTip">
    <string>Reset wallet</string>
   </property>
  </action>
  <action name="m_importSecretKeysAction">
   <property name="text">
    <string>Import spend/view key</string>
   </property>
  </action>
  <action name="m_seedAction">
   <property name="text">
    <string>Import mnemonic seed</string>
   </property>
  </action>
  <action name="m_ConnectionAction">
   <property name="text">
    <string>Connection settings</string>
   </property>
  </action>
  <action name="m_optimizeAction">
   <property name="text">
    <string>Optimize wallet</string>
   </property>
  </action>
  <action name="m_viewWalletAction">
   <property name="text">
    <string>Import view wallet</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::OverviewFrame</class>
   <extends>QFrame</extends>
   <header>gui/OverviewFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::SendFrame</class>
   <extends>QFrame</extends>
   <header>gui/SendFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::TransactionsFrame</class>
   <extends>QFrame</extends>
   <header>gui/TransactionsFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::AddressBookFrame</class>
   <extends>QFrame</extends>
   <header>gui/AddressBookFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::ReceiveFrame</class>
   <extends>QFrame</extends>
   <header>gui/ReceiveFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::MessagesFrame</class>
   <extends>QFrame</extends>
   <header>gui/MessagesFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::SendMessageFrame</class>
   <extends>QFrame</extends>
   <header>gui/SendMessageFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::WelcomeFrame</class>
   <extends>QFrame</extends>
   <header>gui/WelcomeFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::DepositsFrame</class>
   <extends>QFrame</extends>
   <header>gui/DepositsFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::BankingFrame</class>
   <extends>QFrame</extends>
   <header>gui/BankingFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::BankingFrame2</class>
   <extends>QFrame</extends>
   <header>gui/BankingFrame2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_overviewAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_overviewFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_sendFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_receiveAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_receiveFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_transactionsAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_transactionsFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_addressBookFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_encryptWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryptWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_changePasswordAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryptWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutQtAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backupWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>backupWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutCryptonoteAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_startOnLoginAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setStartOnLogin(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_minimizeToTrayAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMinimizeToTray(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeToTrayAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setCloseToTray(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_messagesAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_messagesFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendMessageAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_sendMessageFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_importKeyAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_depositsAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_depositsFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bankingAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_bankingFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_resetAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_importSecretKeysAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importsecretkeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_seedAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ConnectionAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>nodeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_optimizeAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>optimizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_viewWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importTracking()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createWallet()</slot>
  <slot>openWallet()</slot>
  <slot>encryptWallet()</slot>
  <slot>optimizeClicked()</slot>
  <slot>aboutQt()</slot>
  <slot>backupWallet()</slot>
  <slot>about()</slot>
  <slot>setStartOnLogin(bool)</slot>
  <slot>setMinimizeToTray(bool)</slot>
  <slot>setCloseToTray(bool)</slot>
  <slot>importKey()</slot>
  <slot>importSeed()</slot>
  <slot>importTracking()</slot>
  <slot>nodeSettings()</slot>
  <slot>importsecretkeys()</slot>
  <slot>resetWallet()</slot>
 </slots>
</ui>
